object-manipulation-2/src/invert.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-unused-vars */
+
+/* Creates an object composed of the inverted keys and values of source.
+invert({ age: NaN, occupation: 'programmer', language: 'JavaScript' })
+// -> { NaN: 'age', programmer: 'occupation', JavaScript: 'language' }
+invert({ accountId: 'axbxcx', amount: 1000, type: 'withdrawal' })
+// -> { axbxcx: 'accountId', '1000': 'amount', withdrawal: 'type' }
+invert({ name: 'ada', type: 'cat', breed: 'bengal', age: 1.5 })
+// -> { ada: 'name', cat: 'type', bengal: 'breed', '1.5': 'age' }
+invert({})
+// -> {}
+*/
+
+const invert = source => {
+  const obj = source;
+
+  const sourceProp = [];
+
+  const sourceValue = [];
+
+  const newObj = {};
+
+  for (const property in obj) {
+    sourceProp.push(property);
+  }
```

Comment:
You can do all of the modification to `newObj` here. No arrays required.

Reply:
revised!